packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -2237,190 +2250,19 @@ def _get_unanswered_questions(
         response = yield from self.get_subgraph_result(
             query=OPEN_FPMM_QUERY.substitute(
                 creator=creator,
-                first=self.params.questions_to_close_batch_size,
+                first=1000,
```

Comment:
Why this change?

Reply:
Because 1000 is the maximum value returned by TheGraph (more than enough to collect all the expected unanswered questions), and `questions_to_close_batch_size` is expected to be a small number (in the range of 1-5). If these questions cannot be closed (e.g., mech returns error for them), there will be the problem that no more questions will be selected from the set of pending questions to close.

Reply:
Makes sense, can we add a constant though?

Reply:
I'm including it as part of the TheGraph query string, as it happens for the rest of the queries for this service.